### DISCUSSION TITLE
[improvement] Clipboard and Trash on dock

### DISCUSSION THREAD

**Discussion by LeCharlatan** [linuxdeepin/developer-center#2948] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hi,

I like Deepin a lot but it misses some features like a clipboard manager on dock and the trash icon when use extended dock;.

Thanks for good job :+1:

**Comment by atar-axis** [comment]
A trash icon is already implemented, right?
So what is missing is the clipboard - I totally agree with you (even if CTRL + ALT + V) does open the clipboard manager too.

Why don't you give it a try? Here are some examples and introductions:
https://www.deepin.org/en/developer-community/dock-plugin/
https://github.com/v-svg/dde-dock-clock
https://github.com/CareF/deepin-dock-plugin-weather

**Comment by meiyixiang** [comment]
You can upgrade to the latest version to get a feel for it
